.github/workflows/workflow.yml

```diff
@@ -0,0 +1,42 @@
+name: testing
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
Should we test on ubuntu only?

Reply:
I think it's fine - memestra is OS agnostic.